packages/protocol/test/common/linkedlist.ts

```diff
@@ -1,13 +1,8 @@
 import { assertRevert } from '@celo/protocol/lib/test-utils'
-// import BigNumber from 'bignumber.js'
 import { LinkedListTestContract, LinkedListTestInstance } from 'types'
 
 const LinkedListTest: LinkedListTestContract = artifacts.require('LinkedListTest')
 
-// @ts-ignore
-// TODO(mcortesi): Use BN
-LinkedListTest.numberFormat = 'BigNumber'
```

Comment:
Is this not needed anymore? I thought this was needed for some tests in the past.

Reply:
Most tests for linked lists are in addresssortedlinkedlistwithmedian.ts